doc/classes/EditorSettings.xml

```diff
@@ -656,6 +656,9 @@
 			Expanding main editor window content to the title, if supported by [DisplayServer]. See [constant DisplayServer.WINDOW_FLAG_EXTEND_TO_TITLE].
 			Specific to the macOS platform.
 		</member>
+		<member name="interface/editor/font_allow_msdf" type="bool" setter="" getter="">
+			If set to [code]true[/code], MSDF font rendering will be used for visual shader graph editor.
```

Comment:
The setting has a generic name so it seems a bit weird to specifically mention the visual shader graph here. Is that the only editor plugin that would make sure of this being enabled>

Reply:
The font itself is part of the main editor theme, but it is the only part of the editor currently using MSDF font.